static/app/views/replays/detail/trace/trace.tsx

```diff
@@ -215,4 +215,4 @@ const TraceViewWaterfallWrapper = styled('div')`
   height: 100%;
 `;
 
-export default Trace;
+export default React.memo(Trace);
```

Comment:
This is not/should not be needed

Reply:
Removed it. Found a way to pass down the `replay`, without having to use `useReplayContext` in parent component.